File: Pratica 4 - Estrutura de Dados SImples/cpp_logic/exemplo1.cpp
#include <iostream>
#include <vector>

int main() {
    //Em C++, usamos o 'std::vector' para uma lista dinâmica
    std::vector<int> quadrados;

    //Laço 'for' para percorrer os números de 15 a 200
    for (int i = 15; i <= 200; i++) {
        int quadrado = i * i;
        quadrados.push_back(quadrado);
    }

    //Apresentar os resultados usando p laço 'for'
    std::cout << "Quadrados dos números de 15 a 200:" << std::endl;
    for (int i = 0; i < quadrados.size(); i++) {
        std::cout << "O quadrado do número " << (i + 15) << " é " << quadrados[i] << std::endl;
    }

    return 0;
}

File: Pratica 4 - Estrutura de Dados SImples/java_logic/Exemplo1.java
import java.util.ArrayList;

public class Exemplo1 {
    public static void main(String[] args) {
        //Em Java, usamos a 'ArrayList' para criar uma lista dinâmica
        ArrayList<Integer> quadrados = new ArrayList<>();

        //Laço 'for' para percorrer os números de 15 a 200
        for (int i = 15; i <= 200; i++) {
            int quadrado = i * i;
            quadrados.add(i * i);
        }

        //Apresentar os resultados usando outro laço 'for'
        System.out.println("Quadrados dos números de 15 a 200:");
        for (int i = 0; i < quadrados.size(); i++) {
            System.out.println("O quadrado do número " + (i + 15) + " é " + quadrados.get(i));
        }
    }
}